lms/templates/help_modal.html

```diff
@@ -208,7 +202,7 @@
     cycle_modal_tab("#feedback_submit", "#feedback_form_wrapper .close-modal");
     cycle_modal_tab("#feedback-faq-link", "#feedback_success_wrapper .close-modal");
 % else:
-    cycle_modal_tab("#howto-enroll-link", "#help_wrapper .close-modal");
+    cycle_modal_tab("#last-link", "#help_wrapper .close-modal");
```

Comment:
cycle_modal_tab makes sure that when you reach `#last-link` it cycles back to the top of the modal. So there is no CSS involved here. Do you still think I should change `last-link` to be a class?